SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -603,7 +603,7 @@ void USpatialActorChannel::OnReserveEntityIdResponse(const Worker_ReserveEntityI
 	if (Op.status_code != WORKER_STATUS_CODE_SUCCESS)
 	{
 		UE_LOG(LogSpatialGDKActorChannel, Error, TEXT("Failed to reserve entity id. Reason: %s"), UTF8_TO_TCHAR(Op.message));
-		// TODO: From now on, this actor channel will be useless. We need better error handling, or a retry mechanism here.
```

Comment:
Don't we want to wait a bit before retrying?

Reply:
Also worth adding a retry for the CreateEntity failure here also?

Reply:
I don't think waiting is necessary. The most common case of failing to reserve is because of a timeout. I can add the case for CreateEntity.

Reply:
Yeah okay, perhaps timeout isn't necessary currently. We can revisit if there are performance implications.